mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -4604,6 +4604,26 @@ struct XdlopsGemmV2RewritePattern
     int64_t NPerWave =
         op->getAttr("n_per_wave").template dyn_cast<IntegerAttr>().getInt();
 
+    // Obtain coordinate transforms for Matrix A and B.
```

Comment:
This is for transforming 1D LDS to matrix using subview?

Reply:
yes